libs/components/ag-grid/package.json

```diff
@@ -30,8 +30,8 @@
     "@skyux/lookup": "0.0.0-PLACEHOLDER",
     "@skyux/popovers": "0.0.0-PLACEHOLDER",
     "@skyux/theme": "0.0.0-PLACEHOLDER",
-    "ag-grid-angular": "^27.2.0",
-    "ag-grid-community": "^27.2.0"
+    "ag-grid-angular": "^28.2.0",
+    "ag-grid-community": "^28.2.0"
```

Comment:
These ranges are also listed in `libs/components/packages/package.json` `updateGroup`. Might need to update there, too.

Reply:
Good catch.